### PR TITLE
refactor: invert validation with serialization

### PR DESCRIPTION
**Pull request by brainrepo**
closes: #308

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]
LGTM.

File: slides.md [hunk 1]
```diff
@@ -574,82 +580,71 @@ All files |        0 |        0 |        0 |        0 |                   |
...
+https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/
 ---
 
 # Step 6: Exercise 💻
 
-- Create and register a `POST /login` route in `routes/login.js`
+- Validate the response in the users route using a schema:
 
-- Validate the body of the request to ensure it is a JSON object containing two required string properties: `username` and `password`
+  - Created with [`fluent-json-schema`](https://github.com/fastify/fluent-json-schema)
```
Comment (mcollina) [inline slides.md:596]: This explanation should be moved to the validation step.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: slides.md (hunk 1 above)
Comment (brainrepo) [reply slides.md:596]: @mcollina Here The JSON Schema is used to validate the response, before this pr, the validation of the response was done in the serialization step. Should I merge with the validation step? In this case, moving the response validation, our serialization step will be empty.

Comment (mcollina) [reply slides.md:596]: What I'm referring to is fluent-json-schema. This should be introduced earlier in the slides as it is used before.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**